### PR TITLE
include Bombardier Global 5000 aircraft

### PR DESCRIPTION
**Pull request by gasilva**
I implemented a new model for the Global 5000 aircraft based on open literature paper and a run of Aeromatic. I changed some points in the file to reflect better the aerodynamics.

### CODE REVIEW

**Comment by bcoconni** [comment]
Thank for the pull request (PR) @gasilva ! :+1: 

Unfortunately it seems that your submission incidentally triggered an [issue that is existing in the `BR710.xml` file](https://github.com/JSBSim-Team/jsbsim/runs/2694806122?check_suite_focus=true#step:12:104):
```text
In file ./../../../engine/BR710.xml: line 15
XML parse error: not well-formed (invalid token)
```
The fix is quite simple, line 15 just need to be replaced by a blank line since `expat` seems to reject lines of dashes:
https://github.com/JSBSim-Team/jsbsim/blob/6e167e49606d2355139f31b34cbde6d9cfa4340c/engine/BR710.xml#L14-L16

This error was existing in JSBSim prior to your PR but if you don't mind we could hit two birds with one stone if you would include the fix to `BR710.xml` in your PR.

Thanks !

**Comment by bcoconni** [comment]
> The original file of BR710 has an error.
It is not a problem of my PR.

Yes, that is no fault of yours. The error was indeed lying in JSBSim for years.
> Do you what to me to correct it?

Yes please ! That would be much appreciated.

Just fix the file `BR710.xml` with a new commit in your branch and GitHub will automatically update the PR accordingly.

**Comment by gasilva** [comment]
I made one more commit pushed ... Now it is running.

File: aircraft/global5000/scripts/trim-cruise.xml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    
+      Modified by:
+    Guilherme Lima da Silca
+    ATS aerothermalsolutions.co 
+    aerospace@ats4i.com 
+    May 2021
+    
+  </description>          
+
+  <use aircraft="global5000" initialize="scripts/airborne"/>
```
Comment (bcoconni) [inline aircraft/global5000/scripts/trim-cruise.xml:25]:
```suggestion
  <use aircraft="global5000" initialize="airborne"/>
```
Initial conditions files should be located in the folder `aircraft/global5000` to keep consistence with other aircraft.
Please update the line above as suggested and move the file `airborne.xml` from `scripts/` to `aircraft/global5000`
```bash
> git mv scripts/airborne.xml aircraft/global5000/airborne.xml
```

File: aircraft/global5000/scripts/trim-cruise.xml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+    <event name="Trim">
+      <condition> simulation/sim-time-sec ge 1000.0 </condition>
+	  <set name="ic/vc-kts"  value="157.55"/>
+	  <set name="ic/h-sl-ft" value="30000.0"/>
+      <set name="simulation/do_simple_trim" value="1"/>
+    </event>
+
+    <event name="Trim">
+      <condition> simulation/sim-time-sec ge 2000.0 </condition>
```
Comment (bcoconni) [inline aircraft/global5000/scripts/trim-cruise.xml:50]: Same as above, please reduce the waiting time:
```suggestion
      <condition> simulation/sim-time-sec ge 2.0 </condition>
```

File: aircraft/global5000/scripts/trim-cruise.xml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+	  <set name="ic/vc-kts"  value="153.55"/>
+	  <set name="ic/h-sl-ft" value="30000.0"/>
+      <set name="simulation/do_simple_trim" value="1"/>
+    </event>
+
+    <event name="Trim">
+      <condition> simulation/sim-time-sec ge 3000.0 </condition>
```
Comment (bcoconni) [inline aircraft/global5000/scripts/trim-cruise.xml:57]: Same as previously, please reduce the waiting time:
```suggestion
      <condition> simulation/sim-time-sec ge 3.0 </condition>
```

**Review by bcoconni — CHANGES_REQUESTED** [review]
Thanks for the PR.

Please consider addressing the suggested changes:
- Move the file `airborne.xml` from `scripts/` to `aircraft/global5000`
- Reduce the interval times between the trims in order to reduce the execution time of your script (see the 3 places of concern in the code review).

**Comment by gasilva** [comment]
here it goes

**Comment by seanmcleod** [comment]
@gasilva did you see my query regarding the double lift?

File: aircraft/global5000/global5000.xml [hunk 4]
```diff
@@ -0,0 +1,953 @@
...
+            <independentVar lookup="row">aero/alpha-rad</independentVar>
+            <tableData>
+              -0.20 -0.880
+               0.00  0.000
+               0.23  1.000
+               0.60  0.880
+            </tableData>
+          </table>
+      </product>
+    </function>
```
Comment (bcoconni) [inline aircraft/global5000/global5000.xml:604]: As @seanmcleod mentioned this `<function>` seems to be a duplication of the function `aero/coefficient/CLalpha` just above. The result of this duplication is that the lift will be twice as expected.
```suggestion
```

Comment (gasilva) [reply aircraft/global5000/global5000.xml:602]: It is my fault...one is lift an the other is supposed to be CL only...

**Comment by gasilva** [comment]
just checkin the value but I can remove it to make equal do other aircraft...

**Comment by gasilva** [comment]
fixed

**Comment by bcoconni** [comment]
Your PR is now merged.
Thanks for your contribution !

**Comment by gasilva** [comment]
You are welcome

**Review by seanmcleod — COMMENTED** [review]
> just checkin the value but I can remove it to make equal to other aircraft...

Okay, but you've removed the references to the change in lift due to ground-effect, spoilers, speed brakes.

**Comment by gasilva** [comment]
Yes...but it was added after your suggestions.
I kept the original from Aeromatic.

Guilherme A. Lima da Silva, Dr.
CEO
Av. Pompéia, 634 cj. 113 - São Paulo - SP - 05022-000
ats4i.com.br
<https://signaltk.online/click?redirect=http%3A%2F%2Fwww.ats4i.com.br&dID=1622379925642&linkName=ats4i.com.br>



Em sex., 28 de mai. de 2021 às 15:22, Sean McLeod ***@***.***>
escreveu:

**Pull request merged**